integration/tests/iam_v2.sh

```diff
@@ -32,14 +32,25 @@ hab_curl() {
 
 do_test_deploy() {
     log_info "Adding v1 policy (that should be migrated)"
-    local token
+    local token output
     token=$(chef-automate admin-token)
     hab_curl --fail -s -k -H "api-token: $token" \
       -d "$(jo subjects="$(jo -a user:local:alice team:local:ops)" action=read resource=auth:users)" \
       https://localhost/api/v0/auth/policies
 
     log_info "run chef-automate iam upgrade-to-v2"
-    chef-automate iam upgrade-to-v2 || return 1
+    if ! output=$(chef-automate iam upgrade-to-v2); then
```

Comment:
Will this also catch `chef-automate iam upgrade-to-v2 --beta2.1`? (since we could possibly go from v1 -> v2.1, we'd need that)

Reply:
Don't think so -- it's the same migration code.